modules/seqexec/web/shared/src/main/scala/seqexec/web/model/boopickle/ModelBooPicklers.scala

```diff
@@ -114,7 +115,7 @@ trait ModelBooPicklers extends GemModelBooPicklers {
     .addConcreteType[BatchCommandState.Run]
     .addConcreteType[BatchCommandState.Stop.type]
 
-  implicit val executionQueuePickler = generatePickler[ExecutionQueue]
+  implicit val executionQueuePickler = generatePickler[ExecutionQueueView]
```

Comment:
Oh I missed this

Reply:
we still need a explicit boopickler for all fields, like `BatchExecState`

Reply:
Done.